From 63f18a5adfbfa1f62427f02e347a1e0942d4d615 Mon Sep 17 00:00:00 2001
From: Sitaram Chamarty <sitaram@atc.tcs.com>
Date: Sat, 10 Sep 2011 18:57:42 +0530
Subject: [PATCH] fix a rather large typo/thinko in 1006eba

If you look at this change, and the same lines in 1006eba, you can
easily see what I missed...

diff --git a/src/gl-compile-conf b/src/gl-compile-conf
index 2c4ac82..6521215 100755
--- a/src/gl-compile-conf
+++ b/src/gl-compile-conf
@@ -267,7 +267,7 @@ sub parse_conf_line
     elsif ($line =~ /^include "(.+)"/)
     {
         my $include_glob = $1;
-        for my $file (glob($include_glob =~ m(^/) ? $include_glob : "conf/$include_glob")) {
+        for my $file (glob($include_glob =~ m(^/) ? $include_glob : "$GL_ADMINDIR/conf/$include_glob")) {
             die "$ABRT $fragment attempting to include configuration\n" if $fragment ne 'master';
             die "$ABRT included file not found: '$file'\n" unless -f $file;
 
